prow/sidecar/run.go

```diff
@@ -121,13 +122,40 @@ func (o Options) Run() error {
 		aborted = returnCode == 130
 	}
 
-	return o.doUpload(spec, passed, aborted)
+	metadataFile := o.WrapperOptions.MetadataFile
+	if _, err := os.Stat(metadataFile); err != nil {
+		if !os.IsNotExist(err) {
+			logrus.WithError(err).Errorf("Failed to stat %s", metadataFile)
```

Comment:
This should be an info or warn level log, not error level. It is not an error if the job doesn't specify metadata.json.

Reply:
this is error on a None-NotExist error so I think error is correct?

Reply:
Yes, thats right, my mistake.